cmd/kots/cli/install.go

```diff
@@ -445,10 +450,15 @@ func InstallCmd() *cobra.Command {
 				case storetypes.VersionPendingPreflight:
 					log.ActionWithSpinner("Waiting for preflight checks to complete")
 					if err := ValidatePreflightStatus(deployOptions, authSlug, apiEndpoint); err != nil {
-						log.FinishSpinnerWithError()
-						return errors.Wrap(err, "failed to validate preflight results")
+						log.FinishSpinner()
```

Comment:
to keep the old behavior where it will show a red X instead of a green checkmark if preflights do not pass
```suggestion
						log.FinishSpinnerWithError()
```